python/cuml/benchmark/algorithms.py

```diff
@@ -340,6 +342,23 @@ def all_algorithms():
             accuracy_function=_treelite_fil_accuracy_score,
             bench_func=predict,
         ),
+        AlgorithmPair(
+            treelite if has_treelite() else None,
+            cuml.ForestInference,
+            shared_args=dict(n_estimators=10, max_leaf_nodes=2**10),
+            cuml_args=dict(
+                fil_algo="AUTO",
+                output_class=False,
+                threshold=0.5,
+                storage_type="SPARSE",
+            ),
+            name="FIL-SKL",
```

Comment:
Maybe `Sparse-FIL-SKL`? to be clear that this is sparse?

Reply:
That's a bit too long, I prefer shorter names.

Reply:
I don't think the current name conveys that this is sparse specifically. Sparse-FIL could also work.

Reply:
Done.